tests/foreman/virtwho/ui/test_esx.py

```diff
@@ -731,3 +731,51 @@ def test_positive_last_checkin_status(self, default_org, form_data, session):
             )
             session.virtwho_configure.delete(name)
             assert not session.virtwho_configure.search(name)
+
+    @pytest.mark.tier2
+    def test_positive_remove_env_option(self, default_org, form_data, target_sat, session):
+        """remove option 'env=' from the virt-who configuration file and without any error
+
+        :id: 4503985c-8cf5-455c-855f-73dc7645ffe9
+
+        :expectedresults:
+            the option "env=" should be removed from etc/virt-who.d/virt-who.conf
+            /var/log/messages should not display warning message
+
+        :CaseLevel: Integration
+
+        :customerscenario: true
+
+        :CaseImportance: Medium
+
+        :BZ: 1834897
+
+        :customerscenario: true
```

Comment:
Same here

Reply:
Have remove the duplicated :customerscenario: true